code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -88,6 +88,7 @@
 	poison_per_bite = 3
 	var/atom/movable/cocoon_target
 	var/fed = 0
+	var/static/list/consumed_mobs = list() //mobs that have been consumed by nurse spiders to lay eggs

```

Comment:
This is a static list, so that all nurse spiders access it and thus check against it when feeding; cutting it when a spider is destroyed would defeat the purpose.